apps/web/layout/AuthModal.tsx

```diff
@@ -27,15 +28,15 @@ export const AuthModal = ({ isOpen, onClose }: IModal) => {
                             <Button
                                 className="bg-[#4889f4]"
                                 onClick={() => {
-                                    router.push(process.env.NEXT_PUBLIC_API_URL + '/auth/google');
+                                    router.push(ENV.api_base + '/api/auth/google');
```

Comment:
Instead of attaching /api route to all place, just put NEXT_ENV_SERVER_URL as xyz.com/api